gems/sorbet/lib/require_everything.rb

```diff
@@ -106,6 +104,41 @@ def self.patch_kernel
 
   private
 
+  def self.rb_file_paths
+    srb_path = __FILE__
+    matcher = %r{^(.*/gems/)sorbet-([.\d]*)/*}
+    match = matcher.match(srb_path)
+
+    unless match
+      raise "Could not match with sorbet at #{srb_path}"
+    end
+
+    gem_path = match[1]
+    version_number = match[2]
+
+    glob = gem_path + "/sorbet-static-#{version_number}*/libexec/sorbet"
+    unless Dir[glob].any?
+      raise "Could not find sorbet-static executable for sorbet at #{srb_path}"
+    end
+
+    # Should be only one of these per-version
+    sorbet_exec = Dir[glob].first
+    output = `#{sorbet_exec} -p file-table-json --stop-after=parser`
+    # This returns a hash with structure:
+    # { files:
+    #   [
+    #     {
+    #       "strict": ["Ignore"|"False"|"True"|"Strict"|"Strong"],
+    #       "path": "./path/to/file",
+    #       ...
+    #     }
+    #     ...
+    #   ]
+    # }
+    parsed = JSON.parse(output)
+    parsed['files'].reject{|file| file["strict"] == "Ignore"}.filter{|file| File.file?(file["path"])}.map{|file| File.expand_path(file["path"])}
```

Comment:
You'll probably want to reject the stdlib files as well, which will have a strict/sigil value of `Stdlib`.

Reply:
I would go one step further and say to ignore any file that has path that doesn't look like a file path. Specifically, the stdlib paths all have `https://` paths so that jump-to-def opens the RBI in the browser.

Reply:
Oh actually I see that you're already effectively doing this by checking `.file?` on the path itself.

In that case, do you mind writing that motivation into a comment? (It might not be clear to a future reader why `file?` is necessary here.

Reply:
Done in https://github.com/sorbet/sorbet/pull/1351/commits/2e7a56a8d5c00c6d1c49754d8259adbe0e296a6b